worlds/yachtdice/Options.py

```diff
@@ -0,0 +1,331 @@
+    range_start = 1
+    range_end = 5
+    default = 4
+
+
+class NumberRollFragmentsPerRoll(Range):
+    """
+    Rolls can be split into fragments, gathering enough will give you an extra roll.
+    You start with one roll, and there will always be one full roll in the pool.
+    The other three rolls are split into fragments, according to this option.
+    Setting this to 1 fragment per roll just puts "Roll" objects in the pool.
+    """
+
+    display_name = "Number of roll fragments per roll"
+    range_start = 1
+    range_end = 5
+    default = 4
+
+
+class AlternativeCategories(Range):
+    """
+    There are 16 default categories, but there are also 16 alternative categories.
+    These alternative categories can be randomly selected to replace the default categories.
+    They are a little strange, but can give a fun new experience.
+    In the game, you can hover over categories to check what they do.
+    How many alternative categories would you like to see in your game?
+    """
+
+    display_name = "Number of alternative categories"
+    range_start = 0
+    range_end = 16
+    default = 0
+
+
+class ChanceOfDice(Range):
+    """
+    The item pool is always filled in such a way that you can reach a score of 1000.
+    Extra progression items are added that will help you on your quest.
+    You can set the weight for each extra progressive item in the following options.
+
+    Of course, more dice = more points!
+    """
+
+    display_name = "Weight of adding Dice"
+    range_start = 0
+    range_end = 100
+    default = 5
+
+
+class ChanceOfRoll(Range):
+    """
+    With more rolls, you will be able to reach higher scores.
+    """
+
+    display_name = "Weight of adding Roll"
+    range_start = 0
+    range_end = 100
+    default = 20
+
+
+class ChanceOfFixedScoreMultiplier(Range):
+    """
+    Getting a Fixed Score Multiplier will boost all future scores by 10%.
+    """
+
+    display_name = "Weight of adding Fixed Score Multiplier"
+    range_start = 0
+    range_end = 100
+    default = 30
+
+
+class ChanceOfStepScoreMultiplier(Range):
+    """
+    The Step Score Multiplier boosts your multiplier after every roll by 1%, and resets on sheet reset.
+    So, keep high scoring categories for later to get the most out of them.
+    By default, this item is not included. It is fun however, you just need to know the above strategy.
+    """
+
+    display_name = "Weight of adding Step Score Multiplier"
+    range_start = 0
+    range_end = 100
+    default = 0
+
+
+class ChanceOfDoubleCategory(Range):
+    """
+    This option allows categories to appear multiple times.
+    Each time you get a category after the first, its score value gets doubled.
+    """
+
+    display_name = "Weight of adding Category copy"
+    range_start = 0
+    range_end = 100
+    default = 50
+
+
+class ChanceOfPoints(Range):
+    """
+    Getting points gives you... points. You can get 1 point, 10 points, and even 100 points.
```

Comment:
Unclear what this option does. Could the description be rephrased or clarified?

Reply:
Thank you for the feedback!
I've resolved the other comments (changed automatically now defines what it changes, and I've rephrased the questions in the yaml, I agree on those).

I've changed the description of this option to:
    Are you tired of rolling dice countless times and tallying up points one by one, all by yourself? 
    Worry not, as this option will simply add some points items to the item pool! 
    And getting one of these points items gives you... points!
    Imagine how nice it would be to find tons of them. Or even better, having others find them FOR you!
I think it's funny _and_ clear this way. Do you agree (with the fact that it's clear 😄)?